infinitory/errors.py

```diff
@@ -0,0 +1,137 @@
+import os
+import pickle
+import sys
+import time
+
+from datetime import datetime
+
+import logging
+
+
+class ErrorParser(object):
+    def __init__(self):
+        self._reports = dict()
+        self._all_errors = []
+        self._unique_errors = []
+
+        self._logger = logging.getLogger()
+
+    def delete_report_cache(self):
+        cache_path = './cache/'
+        for f in os.listdir(cache_path):
```

Comment:
Could you change this from `f` to `file_name` or something like that? I'd like to keep variables descriptive as much as possible.

Reply:
Will do